.github/workflows/test.yml

```diff
@@ -87,16 +91,20 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 1
+    - name: Install cosign binary
+      uses: sigstore/cosign-installer@v1.3.1
+      with:
+        cosign-release: 'v1.3.1'
     - name: "Register QEMU (tonistiigi/binfmt)"
       run: docker run --privileged --rm tonistiigi/binfmt --install all
     - name: "Prepare (network driver=slirp4netns, port driver=builtin)"
       run: DOCKER_BUILDKIT=1 docker build -t test-integration-rootless --target test-integration-rootless --build-arg CONTAINERD_VERSION=${CONTAINERD_VERSION} .
     - name: "Test    (network driver=slirp4netns, port driver=builtin)"
-      run: docker run -t --rm --privileged test-integration-rootless
+      run: docker run -t --rm --privileged -v $HOME/.cosign/cosign:/usr/local/bin/cosign test-integration-rootless
     - name: "Prepare (network driver=slirp4netns, port driver=slirp4netns)"
       run: DOCKER_BUILDKIT=1 docker build -t test-integration-rootless-port-slirp4netns --target test-integration-rootless-port-slirp4netns --build-arg CONTAINERD_VERSION=${CONTAINERD_VERSION} .
     - name: "Test    (network driver=slirp4netns, port driver=slirp4netns)"
-      run: docker run -t --rm --privileged test-integration-rootless-port-slirp4netns
+      run: docker run -t --rm --privileged -v $HOME/.cosign/cosign:/usr/local/bin/cosign test-integration-rootless-port-slirp4netns
```

Comment:
Yes you are right, but actually, In the beginning, I thought the same thing, then I didn't want to add an additional binary to the nerdctl container image because I assumed that you do not want this either. But if there is not a problem, I can do it gladly.

https://github.com/goreleaser/goreleaser/blob/f9687b482ad00c9176244e9d76f600c765d6ec90/Dockerfile#L14

Reply:
cosign binary should be added in Dockerfile.

Reply:
ok cool, let me do it.

Reply:
I added this to the 89th line, but seems not working, because I saw an error in the tests "cosign executable not found", @AkihiroSuda

Reply:
I moved my cosign binary into the `/out/bin/cosign`, I think it'll work.